### PR TITLE
Add Madara Template (and one Madara source) 

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
A lot of sources (as shown here: https://github.com/inorichi/tachiyomi-extensions/blob/dfb7680b1911a895245987d33bedd8cc5b9fad68/src/all/madara/src/eu/kanade/tachiyomi/extension/all/madara/MadaraFactory.kt#L26-L122), could be added this way.

Right now, only has a minimal number of functions, will add the rest later (along with more Madara sources probably).

### CODE REVIEW

**Comment by ConradWeiser** [comment]
- Return objects seem to be wrapped correctly
- Passes preliminary test cases in a timely manner
- Bundles are verified to contain both the Madara and Source abstract classes

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "axios": "^0.19.2",
     "browserify": "^16.5.1",
     "cheerio": "^1.0.0-rc.3",
-    "gulp": "^4.0.2",
+    "gulp-cli": "^2.2.0",
```
Comment (ConradWeiser) [inline package.json:18]: Is this NPM package required? Can this be removed?

File: src/sources/IsekaiScan/IsekaiScan.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { LanguageCode } from '../../models/Languages/Languages'
+
+
+export class IsekaiScan extends Madara {
+
+    constructor(cheerio: CheerioAPI) {
+        super(cheerio)
+    }
+
+    get version(): string { return '1.0.0' }
```
Comment (ConradWeiser) [inline src/sources/IsekaiScan/IsekaiScan.ts:11]: I'd start with version 1.0.0 after you've done extensive testing and are sure that it's stable. Perhaps consider a lower number until you know it all works

File: package.json (hunk 1 above)
Comment (AbdullahM0hamed) [reply package.json]: Not really. My mistake. Didn't know it was even added😭😭😭

**Pull request merged**